### PR TITLE
Add reworked models of the Archons and new Archon model

### PR DESCRIPTION
**Pull request by GefullteTaubenbrust2**
## Summary
Adds new models for the existing Archons, as well as providing assets for an entirely new Archon.

comparison

## Save File
This save file can be used to view the new sprites (it is a bit tedious to do otherwise):
[The Test.txt](https://github.com/endless-sky/endless-sky/files/12411745/The.Test.txt)

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/146
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/369

### CODE REVIEW

File: copyright [hunk 1]
```diff
@@ -1605,6 +1600,15 @@ Files:
...
 License: CC-BY-SA-4.0
 
+Files:
+ images/effect/archonteleport/*
+ images/effect/archonbteleport/*
+ images/effect/archoncteleport/*
+ images/effect/archondteleport/*
+Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
+License: CC-BY-SA-4.0
+Comment: Derived from works by @Karirawri (under the same license).
```
Comment (beccabunny) [inline copyright:1610]: If they're derived from Karirawri, and Karirawri's are derived from MZ and I, should this be derived from us two as well?

Comment (GefullteTaubenbrust2) [reply copyright:1610]: I have no idea tbh

**Comment by Amazinite** [comment]
The removal of the bulbs under the main body of the Archon result in the silhouette appearing thinner than they were before. I prefer the older, chunkier look, as the larger visual size gives the Archons more of a sense of presence; the existing ones seem more imposing to me.
image

**Comment by lumbar527** [comment]
Yeah, the remodel _does_ look a little skinny. I think it could just be fixed by making the lower part a little wider, though. Or maybe just thickening the whole thing.

**Comment by Arachi-Lover** [comment]
Could add the chunky bits below it back in again too maybe.

**Pull request closed without merging**